app/src/main/kotlin/com/simplemobiletools/calculator/activities/MoneyActivity.kt

```diff
@@ -55,6 +66,8 @@ class MoneyActivity : SimpleActivity(), Calculator , TaxCalculator {
 
         AutofitHelper.create(result)
 
+        JsonTask().execute("https://api.fixer.io/latest?base=CAD")
```

Comment:
Are only CAD conversions supported?

Reply:
All conversions are supported by performing a double conversion, e.g. EUR -> USD: EUR*(CAD/EUR)*(USD/CAD)

Reply:
That seems super convoluted to me why not just do it straight away instead of two queries ?

Reply:
Only one API query is made to get the currency conversion ratios against the base CAD. A bit of math is performed in CurrencyConversionOperation::getRate() that uses this data to support conversion between non-CAD currencies.